Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM starefossen/github-pages
+FROM starefossen/github-pages:177
```

Comment:
I think this needs to be updated in https://github.com/docker/docker.github.io/tree/publish-tools

Reply:
If you're bumping the Dockerfile to use GH Pages 177, could you also bump the Gemfile used for local `jekyll build`? - https://github.com/docker/docker.github.io/blob/master/Gemfile

Reply:
I admin I don't fully understand how things are working here. This branch [README.md](https://github.com/docker/docker.github.io/blob/docs-base/README.md#reasons-to-update-this-branch) says:
> Reasons to update this branch
> ...
> Updating the GitHub Pages version used in production to a newer release

... but the branch you mentioned also pulls gh-pages. I guess I'll have two PRs and let the more knowledgeable people decide which one to take.

Reply:
https://github.com/docker/docker.github.io/pull/6159

Reply:
> could you also bump the Gemfile used for local jekyll build? - https://github.com/docker/docker.github.io/blob/master/Gemfile

It's a separate branch so it requires a separate PR. Here: https://github.com/docker/docker.github.io/pull/6167